src/librustc/lib.rs

```diff
@@ -40,7 +40,11 @@
 
 #![recursion_limit="256"]
 
+#[allow(unused_extern_crates)]
```

Comment:
Can you add a comment here pointing to the `Cargo.toml` of rustc_cratesio_shim?

Reply:
Done.